cmd/nodecmd/create_aws.go

```diff
@@ -355,6 +375,29 @@ func AddMonitoringSecurityGroupRule(ec2Svc map[string]*awsAPI.AwsCloud, monitori
 	return nil
 }
 
+func grantAccessToPublicIPViaSecurityGroup(ec2Svc *awsAPI.AwsCloud, publicIP, securityGroupName, region string) error {
+	securityGroupExists, sg, err := ec2Svc.CheckSecurityGroupExists(securityGroupName)
+	if err != nil {
+		return err
+	}
+	if !securityGroupExists {
+		return fmt.Errorf("security group %s doesn't exist in region %s", securityGroupName, region)
```

Comment:
`region` is only used for error msg, is it worth passing it only for this

Reply:
anyway, it's not that important vs not using this function at all as https://github.com/ava-labs/avalanche-cli/pull/1495/files#r1513554306